src/mono/wasm/testassets/BlazorHostedApp/BlazorHosted.Server/ChatHub.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.SignalR;
+
+namespace BlazorHosted.Server.Hubs;
+public class ChatHub : Hub
+{
+    public async Task SendMessage(string message, int sendingThreadId)
+    {
+        TestOutputWriteLine($"Server: receives Message=[{message}] sent by treadID = {sendingThreadId} and sends it back.");
+        string changedMessage = $"{message}-pong";
+        await Clients.All.SendAsync("ReceiveMessage", changedMessage).ConfigureAwait(false);
+    }
+
+    public void Exit(int code)
+    {
+        TestOutputWriteLine($"Received exit code {code} from client.");
+        Environment.Exit(code);
+    }
+
+    public static void TestOutputWriteLine(string message)
+    {
+        Console.WriteLine("TestOutput -> " + message);
```

Comment:
Are we processing these messages as "test output"? The idea for this prefix is to be able to distiguish between "standard app output" and "dedicated message that we can later assert in the test case"